documentation/ci-fix.md

```diff
@@ -49,18 +49,31 @@ Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/st
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
 ## Breaking Change Check
-
+There are two kind of breaking change checks: same version breaking change check and cross version breaking change check. Both uses the OAD (OpenAPI Diff) tool to identify breaking changes between swaggers.
+- The same version breaking change check compares the existing version swaggers with updated swaggers of the same version, so that to identify if there are any updates break the version.
+- The cross version breaking change check usually applies to new API version scenario. It uses swaggers of the latest public version as the baseline, comparing swaggers of new version with the baseline, so that to identify if there are any updates lead to breaking experience cross versions.
```

Comment:
The cross version check ensures that a new api-version is backward compatible with the previous api-version.